frontend/lib/l10n/app_de.arb

```diff
@@ -115,7 +112,7 @@
   "search_filterByCategories": "Nach Kategorien filtern",
   "search_findCloseBy": "In meiner Nähe suchen",
   "search_noAcceptingStoresFound": "Auf diese Suche trifft keine Akzeptanzstelle zu.",
-  "search_searchHint": "Tippen, um zu suchen …",
+  "search_searchHint": "Tippen, um zu suchen...",
```

Comment:
hm basically this could be my issue with screenshottests. Why the points are different?
Please also adjust that in the screenshot test file or they will fail (frontend/ios/ScreenshotTests/)

Reply:
Will revert